Docs/source/running_cpp/parameters.rst

```diff
@@ -1036,7 +1036,7 @@ Numerics and algorithms
     The medium for evaluating the Maxwell solver. Available options are :
```

Comment:
I can make another suggestion for simplification of the user input. This input, `em_solver_medium`, is also not needed. The code could query if any of the macroscopic input parameters are set and if so, use the macroscopic solver, otherwise use the vacuum solver. This would take some code rearrangement since the macroscopic input would always have to be checked.

Reply:
@dpgrote is it okay if I put a pin on this suggestion and make a mini-PR later to add this?

Reply:
No problem. This is just a suggestion and not critical.